### ISSUE TITLE
Allow customizing columns

### ISSUE THREAD

**Issue by phsm** [coderaiser/cloudcmd#146]
* **Version** (`cloudcmd -v`): v8.2.2
* **Node Version** `node -v`: v6.12.0
* **OS** (`uname -a` on Linux): ubuntu 18.04
* **Browser name/version**:  Firefox 58.0.2
* **Used Command Line Parameters**:  cloudcmd --no-console --no-terminal --no-online --no-config-dialog --root /path/to/root
* **Changed Config**: none

Hello,
I have a feature request
Long names are unreadable even on 1920 width monitor (see screenshot). 
The possible solution could be to allow setting which columns to display in config and/or via commandline. 
For example, in my case if columns "owner" and "mode" had been hidden, there was enough space to fit the whole filename.
![2018-02-26-222308_1904x158_scrot](https://user-images.githubusercontent.com/9035611/36693801-1340d1d6-1b45-11e8-99b7-6e581ca335fe.png)

**Comment by coderaiser** [comment]
It is a great idea for a pull request :) and duplicate of https://github.com/coderaiser/cloudcmd/issues/139.

**Comment by phsm** [comment]
i'd be glad to contribute, but I don't know node.js at at all :(

**Comment by coderaiser** [comment]
It is more about `css styling` then `node.js` :)

**Comment by LarkinZero** [comment]
```
.name{width:60%}
.size{float:right}
.date{float:right}
.owner{display:none}
.mode{display:none}
```

![qq20180228-115542](https://user-images.githubusercontent.com/6387162/36769284-57d7ba8c-1c7e-11e8-936d-fb89ac900a5e.png)

**Comment by coderaiser** [comment]
Landed in [v8.3.0](https://github.com/coderaiser/cloudcmd/releases/tag/v8.3.0).
